src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyLineBreakAfter.java

```diff
@@ -26,10 +26,18 @@ else if (x < 0) {
                 else { break; }
                 switch (a)
                 {
-                case 0:
-                    break;
-                default:
-                    break;
+                    case 1:
```

Comment:
Do we have test cases with `{` which do not produce violations. If no please add.

Reply:
done